src/stcal/jump/jump.py

```diff
@@ -319,12 +357,13 @@ def detect_jumps(frames_per_group, data, gdq, pdq, err,
 
 def flag_large_events(gdq, jump_flag, sat_flag, min_sat_area=1,
                       min_jump_area=6,
-                      expand_factor=1.9, use_ellipses=False,
-                      sat_required_snowball=True):
+                      expand_factor=2.0, use_ellipses=False,
```

Comment:
Since these have default values in 'spec' in jwst, they shouldn't also have defaults here

Reply:
This allows test to run without having to specify all the parameters. Also, it allows STCAL to be merged without requiring JWST to have all the new parameters.